pybikes/velobike.py

```diff
@@ -4,14 +4,16 @@
 from .base import BikeShareSystem, BikeShareStation
 from . import utils
 
-
 class Velobike(BikeShareSystem):
 
     sync = True
 
     meta = {
         'system': 'Velobike',
-        'company': 'Velobike.kz, LLP'
+        'company': ['Agency for Physical Culture and Sports of \
+                    the Republic of Kazakhstan', 
+                    'Sovereign Wealth Fund "Samruk-Kazyna" JSC',
+                    'Akimat of Astana', 'Smoove']
```

Comment:
I think we should have `Velobike.kz, LLP` in there too (http://velobike.kz/lease_agreement_en.pdf).

For me, clearest would be:
```
['Velobike.kz, LLP', 'Smoove']
```